pkg/flex/mounter.go

```diff
@@ -53,9 +53,33 @@ func (m *Mounter) Mount(targetPath string, specString string) *Response {
 		return NewFailResponse("Failed to create v3io FUSE container", err)
 	}
 
+	m.createDirs(spec, targetPath)
+
 	return NewSuccessResponse("Successfully mounted")
 }
 
+func (m *Mounter) createDirs(spec Spec, targetPath string) {
```

Comment:
must return an error. the error can be logged at the caller (since it return an error itself).
make sure to follow the pattern of
```
err := something()
if err != nil {
    return err 
}

... the else part is here ...
```

note that usually we use a wrapping error mechanism, but it doesn't exist in flex-fuse so you're off the hook

Reply:
I do not want to return error here. I don't want to have any impact on the main mount flow